src/components/application_manager/src/commands/command_request_impl.cc

```diff
@@ -248,9 +247,11 @@ void CommandRequestImpl::SendResponse(
     current_state_ = kCompleted;
   }
 
-  smart_objects::SmartObjectSPtr result =
-      utils::MakeShared<smart_objects::SmartObject>();
-
+  smart_objects::SmartObjectSPtr result = new smart_objects::SmartObject;
+  if (!result) {
+    LOG4CXX_ERROR(logger_, "Memory allocation failed.");
```

Comment:
@pold500 Please do not use such allocation checks, it is impossible to reach it by unit tests. Please use 'utils::MakeShared' to allocete your SmartObject instead of 'new' operator.